### DISCUSSION TITLE
Vital 100s options?

### DISCUSSION THREAD

**Discussion by ghubg10001** [RaresAil/homebridge-levoit-air-purifier#52] [General]
I get the 100 and 200 operate on different APIs. Just curious if the Vital 100s has any ability to map through or if it's not possible. I tried to find another plugin and there doesn't seem to be one.

**Comment by eoddc5** [comment]
From my understanding it’s the core series that the 100 has different api (or rather anything under the 200 model)

the vital 100s (and vital 200s) are both newer than all the core models, so their stuff should work - just needs support added by @RaresAil

**Reply by RaresAil** [reply]
I will look into it these days, i might need someone with those new air purifiers to test it

**Reply by eoddc5** [reply]
@RaresAil - thanks!

I also made a PR for the main branch, if you could check it out and publish if it looks good

I purchased a Core 300S and Core 600S, both with the PlasmaPro feature, and their models are Core302S and Core 602S

so I just added those into the devicetypes.ts - and I think it will be able to pull them in

**Reply by eoddc5** [reply]
id also be willing to help out with plasma features, coming from this old thread: https://github.com/RaresAil/homebridge-levoit-air-purifier/discussions/30

if you need me to post logs, just say so

**Reply by RaresAil** [reply]
@eoddc5 can you use the last version and post some logs with the vital devices?

**Reply by Dxtros** [reply]
@RaresAil  I posted my logs with my vital 100s on #54 let me know if you need anything else.

**Comment by RaresAil** [comment]
@Jur1nforest 131S is not supported and it will not be because is too old

**Comment by shipham2302** [comment]
Also wait for Vital series (100s and 200s). Just bought it today and hopefully to get it into homekit soon!

**Reply by Ms-Sudo** [reply]
@RaresAil Thanks for all your work. I have the Vital 100S and your plugin works quite well.

Almost everything works:
- Mode: Manual or Auto (in the settings)
- Child Lock: On/Off (in the settings)
- Turning it On/Off
- Display Off (Turning it on somewhat not)

Displays:
- Air Quality Status (on top for the room)
- Filter Condition (in the settings)
- Filter Life (in the settings)
- Serial Nr. (in the settings)
- Model (in the settings)
- Firmware (in the settings)


So far not working yet:
- Fan Speed Control (1-4)
- Turn on Sleep Mode
- Turn on PET Mode

**Comment by Brandawg93** [comment]
#91
